lib/modules/manager/kotlin-script/__fixtures__/no-dependencies.main.kts

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env kotlin
+println("Hello world")
```

Comment:
please inline such small fixtures to reduce IO (which is pretty bad on windows 😉 )